test/yamlAST.test.ts

```diff
@@ -106,4 +107,12 @@ suite('YAML AST', () => {
       
     });
   });
+
+  suite('Flow style', () => {
+    test('should parse flow style map', () => {
+      const content = '{A: , B: b1}';
+      const result: YamlMap = safeLoad(content) as YamlMap;
+      expect(result.mappings[0].value.startPosition).is.greaterThan(0);
```

Comment:
I think we should probably change this to an exact value since I don't think we expect it to change once this PR is merged